src/platform/MpiServer.c

```diff
@@ -0,0 +1,611 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <pthread.h>
+
+#include <PlatformCommon.h>
+#include <MpiServer.h>
```

Comment:
Add pthread.h to PlatformCommon.h. Include only that plus the MpiServer.h here. Without blank line in between.